src/common/selectors.js

```diff
@@ -448,12 +448,17 @@ export const createGetTags = collectionSelectors => {
   return _extendCollectionSelector(getTags, 'tag')
 }
 
-export const createGetLastShutdownTime = () => create(
-  (_, vm) => vm.id,
+export const createGetVmLastShutdownTime = (vmProp, predicate) => create(
+  (_, props) => vmProp(props.vm),
```

Comment:
Do not pass a property getter but directly a selector:

```js
return {
  lastShutdownTime: createGetVmLastShutdownTime((_, props) => props.vm)
}
```

And it would be nice if it has a default value.